apidef/oas/default.go

```diff
@@ -218,6 +221,7 @@ func (s *OAS) importMiddlewares(overRideValues TykExtensionConfigParams) {
 	}
 
 	for path, pathItem := range s.Paths {
+		overRideValues.pathItemHasParameters = len(pathItem.Parameters) > 0
```

Comment:
Can we have this in function scope rather than data model? Seems you just need to pass it to Import(). Are we breaking an API if we change the Import signature?